config/routes.rb

```diff
@@ -24,5 +19,6 @@
         post :login, to: 'doorkeeper/tokens#create'
       end
     end
+    resources :jwt_api_entreprise, only: [:update]
```

Comment:
c'est possible de mettre cette ligne entre `roles`et `users` (ie: après la ligne 10) ? Là c'est isolé et ça floute un peu la lecture

Reply:
Oui je vais faire ça. Perso je n'arrive pas à lire simplement ce fichier de route. Aujourd'hui si je veux une liste des routes disponibles je fais un `rake routes`...

Je me suis noté de refactor ça en définissant chaque routes manuellement pour gagner en clarté, comme on le fait déjà dans SIADE.